samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -330,11 +336,20 @@ SamzaContainer getContainer() {
 
   @VisibleForTesting
   SamzaContainer createSamzaContainer(String processorId, JobModel jobModel) {
+
+    // Creating diagnostics manager and reporter, and wiring it respectively
+    Optional<Tuple2<DiagnosticsManager, MetricsSnapshotReporter>> diagnosticsManagerReporterPair = DiagnosticsUtil.buildDiagnosticsManager(new ApplicationConfig(config).getAppName(), new ApplicationConfig(config).getAppId(), processorId, Optional.empty(), config);
```

Comment:
Why is this appName and the other ones jobName?
Nitpick: extract fields.

Reply:
I thought standalone would require appconfig, replaced with job-config